### PR TITLE
Hydrator-1081 documentation on runtime-arguments for pipelines

### PR DESCRIPTION
**Pull request by gsps1**
JIRA : https://issues.cask.co/browse/HYDRATOR-1081

Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB172-3

Page of interest: http://builds.cask.co/artifact/CDAP-DQB172/shared/build-3/Docs-HTML/4.0.0-SNAPSHOT/en/hydrator-manual/creating-pipelines.html#property-lookups

### CODE REVIEW

**Comment by awholegunch** [comment]
On-hold because I see a typo.

**Comment by awholegunch** [comment]
Docs LGTM. @gsps1 : do you agree?

File: cdap-docs/hydrator-manual/source/creating-pipelines.rst [hunk 1]
```diff
@@ -400,11 +400,20 @@ This order is used so that the most volatile source (the workflow token) takes p
...
 
-  ${source-stream-name}
+  ${stream-name}
   
 and in the runtime arguments set a key-value pair such as::
 
-  source-stream-name: myDemoStream
+  stream-name: myDemoStream
+
+In the Cask Hhydrator UI, macro property look-ups are determined from the configuration and are set as runtime arguments.
```
Comment (gsps1) [inline cdap-docs/hydrator-manual/source/creating-pipelines.rst]: typo here. `Hydrator`

Comment (awholegunch) [reply cdap-docs/hydrator-manual/source/creating-pipelines.rst]: Fixed in last commit.

**Comment by gsps1** [comment]
LGTM, squash and merge.

**Pull request merged**